ftw/theming/extensions.py

```diff
@@ -53,15 +53,21 @@ def find_relative_resource(relpath):
     return path
 
 
+def chunks(l, n):
+    if not l:
+        return []
+    return [l[i:i + n] for i in xrange(0, len(l), n)]
+
+
 def fill_svg(data, fill_css=None, fill_xpath=None):
     doc = etree.fromstring(data)
     expressions = []
 
-    for xpr, color in fill_xpath or ():
+    for xpr, color in chunks(fill_xpath, 2) or ():
```

Comment:
@jone The three pull requests above fix these compatibility issues you mention here.